apps/backend/src/controllers/Question.ts

```diff
@@ -1,8 +1,9 @@
-import { NextFunction, Request, Response } from "express";
+import express, { NextFunction, Request, Response } from "express";

```

Comment:
```suggestion
import { Router,NextFunction, Request, Response } from "express";
```

Prefer specific imports to reduce the size of imports per file.